src/components/v2v3/V2V3Project/ProjectActivity/hooks/ProjectActivity.ts

```diff
@@ -210,45 +237,43 @@ export function useProjectActivity({
     return _where
   }, [projectId, eventFilter])
 
-  const keys: ActivityQueryKey[] = useMemo(() => {
-    let key: ActivityQueryKey | undefined = undefined
+  const keys = useMemo(() => {
+    let _keys = undefined
 
     switch (eventFilter) {
       case 'deployERC20':
-        key = DEPLOYED_ERC20_EVENT_KEY
+        _keys = [DEPLOYED_ERC20_EVENT_KEY]
         break
       case 'pay':
-        key = PAY_EVENT_KEY
+        _keys = [PAY_EVENT_KEY]
         break
       case 'burn':
-        key = BURN_EVENT_KEY
+        _keys = [BURN_EVENT_KEY]
         break
       case 'addToBalance':
-        key = ADD_TO_BALANCE_EVENT_KEY
+        _keys = [ADD_TO_BALANCE_EVENT_KEY]
         break
       case 'projectCreate':
-        key = PROJECT_CREATE_EVENT_KEY
+        _keys = [PROJECT_CREATE_EVENT_KEY]
         break
       case 'redeem':
-        key = REDEEM_EVENT_KEY
+        _keys = [REDEEM_EVENT_KEY]
         break
       case 'distributePayouts':
-        key = DISTRIBUTED_PAYOUTS_EVENT_KEY
+        _keys = [DISTRIBUTED_PAYOUTS_EVENT_KEY]
         break
       case 'distributeTokens':
-        key = DISTRIBUTED_RESERVED_TOKENS_EVENT_KEY
+        _keys = [DISTRIBUTED_RESERVED_TOKENS_EVENT_KEY]
         break
       case 'deployETHERC20ProjectPayer':
-        key = DEPLOYED_PROJECT_PAYER_EVENT_KEY
+        _keys = [DEPLOYED_PROJECT_PAYER_EVENT_KEY]
         break
       case 'configure':
-        key = CONFIGURE_EVENT_KEY
+        _keys = [CONFIGURE_EVENT_KEY, SET_FUND_ACCESS_CONSTRAINTS_EVENT_KEY]
```

Comment:
reorganized this `key = ` pattern so we could make the "configure" filter show both ConfigureEvents and SetFundAccessConstraintsEvents